src/components/Message/MessageSimple/MessageContent.js

```diff
@@ -399,6 +399,21 @@ const MessageContent = (props) => {
 
 MessageContent.themePath = 'message.content';
 
+MessageContent.defaultProps = {
```

Comment:
Without this, props won't be forwarded with default values when you do `<ChildComponent {...props} />`